src/js/game/LevelMVC/LevelBlock.js

```diff
@@ -144,7 +144,9 @@ module.exports = class LevelBlock {
     if (blockType.startsWith("piston")) {
       this.isDestroyable = false;
       this.isConnectedToRedstone = !blockType.startsWith("pistonArm");
-      if (blockType.substring(blockType.length - 2, blockType.length) === "On" || blockType.startsWith("pistonArm")) {
+      if (blockType.substring(blockType.length - 2, blockType.length) === "On" ||
+      blockType.startsWith("pistonArm") ||
+      blockType.substring(blockType.length - 8, blockType.length) === "OnSticky") {
```

Comment:
nit: multiline control statements [should be indented on their own lines](https://github.com/airbnb/javascript#control-statements):

```javascript
if (
  blockType.substring(blockType.length - 2, blockType.length) === "On" ||
  blockType.startsWith("pistonArm") ||
  blockType.substring(blockType.length - 8, blockType.length) === "OnSticky"
) {
```